account/identity.go

```diff
@@ -138,6 +142,28 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
+// Exists returns true|false where an object exists with an identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
+
+	queryStmt, err := m.db.CommonDB().Prepare(fmt.Sprintf(`
```

Comment:
@kwk The sql query is intended to be the most efficient in terms of performance. The same reason applies for the usage of a prepared statement.